float32[] voxel_mean_intensity
float32[] voxel_std_intensity
float32[] voxel_roughness
float32[] voxel_slope